handlebars/handlebars-1.0.0.d.ts

```diff
@@ -0,0 +1,184 @@
+// Type definitions for Handlebars 1.0

```

Comment:
I think you might have to do 2 separate commits to preserve history here. The first commit would be a `git mv` command to rename this file from `handlebars.d.ts` to `handlebars-1.0.0.d.ts`. The 2nd commit would be introducing a new `handlebars.d.ts` with the newest definitions.